docs/troubleshooting.md

```diff
@@ -0,0 +1,9 @@
+## Troubleshooting 
```

Comment:
Looking at this again ... This is a pretty generic title.  Should we have a section for troubleshooting the build?